examples/serial/serial.c

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (C) 2017 Kyle Robbertze <krobbertze@gmail.com>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include <delay.h>
+#include <fx2macros.h>
+#include <fx2regs.h>
+#include <fx2types.h>
+
+#define PD3 0xB3
+#define BAUD 32
```

Comment:
Yes, the asm code sending serial out. https://github.com/timvideos/HDMI2USB-fx2-firmware/pull/18/files#diff-af77cda4c5f8d74a37998c7dec772b7cR42